.github/workflows/workflow.yaml

```diff
@@ -199,7 +199,8 @@ jobs:
           make e2e-verify-release IMG=${{ env.IMAGE_REPO }}:${TAG} USE_LOCAL_IMG=false
 
       - name: Create GitHub release
-        uses: "marvinpinto/action-automatic-releases@v1.1.0"
+        # using latest to fix set-env deprecation
+        uses: "marvinpinto/action-automatic-releases@latest"
```

Comment:
Do we want to be using latest here? Is there a version we should be pinning to to avoid spontaneous breakage?

Reply:
unfortunately, not yet. https://github.com/marvinpinto/action-automatic-releases/releases
we'll want to pin it once it's available.

Reply:
Ah well, thanks for clarifying!